docs/core/toc.yml

```diff
@@ -49,8 +49,9 @@
   - name: Install localized IntelliSense
     href: install/localized-intellisense.md
 - name: Get started
-  href: get-started.md
   items:
+  - name: Get started with C# and the CLI
```

Comment:
This is our very high-level get started. I wouldn't want to add the words CLI in the TOC.

Reply:
On second thought it would be better to keep the name "Get started" for the high-level get started. The current TOC arrangement is confusing because we have two separate TOC sections that list tutorials, one named "Get started" and one named "Tutorials".  It might make more sense to move the tutorials currently under "Get started" to the "Tutorials" section and make "Get started" a top-level leaf node pointing to the high-level get-started.

Reply:
Changed it back to just "Get started", will reserve consolidating the two tutorials sections for a separate PR.